assets/js/navbar.js

```diff
@@ -20,13 +20,22 @@ const controllerNavbar = (function (jQuery) {
       const dropdownToggle = dropdownMenu.find('.dropdown-toggle');
       const dropdownMenuContent = dropdownMenu.find('.dropdown-menu');
 
-      dropdownToggle.on('mouseenter', function () {
-        dropdownMenuContent.addClass('show');
-      })
-
-      dropdownMenu.on('mouseleave', function () {
-        dropdownMenuContent.removeClass('show');
-      })
+      // Toggle dropdown menu when clicking the dropdown toggle
+      dropdownToggle.on('click', function (e) {
+        e.preventDefault(); // Prevent default anchor behavior
+        
+        // Close all other dropdowns
+        jQuery('.dropdown-menu').not(dropdownMenuContent).removeClass('show');
+        
+        dropdownMenuContent.toggleClass('show');
+      });
+
+      // Hide dropdown menu when clicking outside the dropdown menu
+      jQuery(document).on('click', function (e) {
+        if (!dropdownMenu.is(e.target) && dropdownMenu.has(e.target).length === 0 && !dropdownToggle.is(e.target) && dropdownToggle.has(e.target).length === 0) {
+          dropdownMenuContent.removeClass('show');
```

Comment:
If you see `show` exist 3-4 times in the file, would be nice to create a const and re-use everywhere in the file, for example: 
```
const SHOW_CLASS = 'show'; 
```